rwgame/GameBase.cpp

```diff
@@ -105,6 +31,59 @@ GameBase::GameBase(Logger &inlog, int argc, char *argv[]) :
             [this]() {window.hideCursor();});
 }
 
+RWConfig GameBase::build_config(const std::optional<RWArgConfigLayer> &args) {
```

Comment:
I'd stay with camel case for functions/methods names.

Reply:
aight